HW1/task2.py

```diff
@@ -0,0 +1,10 @@
+number = int(input("Введите количество журавликов "))
+person1 = 0
+other = 0
+
+if number%4==0:
+    person1 = int(number/2)
+    other = int(person1/2)
```

Comment:
Проще было бы использовать оператор целочисленного отделения.
```suggestion
    person1 = number // 2
    other = person1 // 2
```